docs/application/vscode-ext/web.md

```diff
@@ -221,35 +212,32 @@ To build project, follow these steps:
 
       ![Building your project](media/tizen_web13.png)
 
-      > **Note**  
-      > 
+      > [!NOTE] 
       > If you have a Tizen device and want to build the application into it, then you must first generate a certificate and add it.
 
-### Exclude Build List
+### Exclude build list
 
 To build the project after removing unwanted files, follow these steps:
 
   1. In VS Code, click **View** > **Command Palette**.
   2. In the input field of the **Command Palette** that appears, type **Tizen Web: Make Exclude Build List** and press **Enter**.
   3. Enter the name of the file to remove. 
   
-      > **Note**
-      >
+      > [!NOTE]
       > Separate the file name with commas and ensure that there is no space between them. For example, file1.txt,file2.txt.
   
       **.buildignore** file is created with the input data in the project folder and the build excludes the input data and generates a packaged `wgt` file.
 
       ![Exclude Build List](media/tizen_web30.png)
 
-### Deploy and Run Application in Emulator
+### Deploy and run your application in emulator
 
 To deploy and run application, follow these steps:
 
 1. In VS Code, click **View** > **Command Palette**.
 2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Emulator-Manager** and press **Enter**.
 
-   > **Note**  
-   >
+   > [!NOTE]  
```

Comment:
Would you please remove 2 spaces after `[!NOTE]`? It is causing line break in the note.
![image](https://user-images.githubusercontent.com/9958374/75840460-9a547200-5e0e-11ea-97f7-b6cc9f5f6ddc.png)